src/Nethermind/Nethermind.Consensus/Processing/BlockProcessor.cs

```diff
@@ -37,12 +39,14 @@ public partial class BlockProcessor(
     IBlockProcessor.IBlockTransactionsExecutor? blockTransactionsExecutor,
     IWorldState? stateProvider,
     IReceiptStorage? receiptStorage,
-    IBlockhashStore? blockHashStore,
+    ITransactionProcessor transactionProcessor,
     IBeaconBlockRootHandler? beaconBlockRootHandler,
+    IBlockhashStore? blockHashStore,
```

Comment:
IMO should be left in same order of parameters

Reply:
@rjnrohit